nixos/modules/virtualisation/containers.nix

```diff
@@ -134,6 +134,9 @@ let
         --setenv HOST_ADDRESS6="$HOST_ADDRESS6" \
         --setenv LOCAL_ADDRESS6="$LOCAL_ADDRESS6" \
         --setenv PATH="$PATH" \
+        ${if cfg . "additionalCapabilities" or null != null then
+          ''--capability="${concatStringsSep " " cfg . "additionalCapabilities"}"'' else ""
```

Comment:
null != null is always false.
People usually use the function `optionalString` from lib to add optional strings.